gulpfile.babel.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+// This gulpfile makes use of new JavaScript features.
+// Babel handles this without us having to do anything. It just works.
+// You can read more about the new JavaScript features here:
+// https://babeljs.io/docs/learn-es2015/
+
+import path from 'path';
+import gulp from 'gulp';
+import del from 'del';
+import runSequence from 'run-sequence';
+// import swPrecache from 'sw-precache';
+import gulpLoadPlugins from 'gulp-load-plugins';
+// import pkg from './package.json';
+import merge from 'merge-stream';
+
+const $ = gulpLoadPlugins();
+
+// Compile and automatically prefix stylesheets
+gulp.task('styles', () => {
+  const AUTOPREFIXER_BROWSERS = [
```

Comment:
This definition isn't currently used. I left out the autoprefixer, because I testing by doing an exact compare of the results from Grunt. My thinking was to make the autoprefixer changes separately in #319.